source/_code.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <button aria-controls="<%= name %>-snippets"
+          aria-expanded="false"
```

Comment:
In the spirit of progressive enhancement, should `aria-controls` and `aria-expanded` be added to the button via JavaScript, once it is available? In fact, should this button be injected via JavaScript?